pkg/admin/deploy_local.go

```diff
@@ -54,8 +57,8 @@ func (d *LocalDeployer) Apply(f func(*LocalDeployer) error) error {
 }
 
 func SetupFLNetwork(d *LocalDeployer) error {
```

Comment:
It could be changed to SetupFLNetworks, in plural, since it creates 2 networks or split in SetupFLNetwork and SetupFLRuntimeNetwork.